demos/src/primary.mustache

```diff
@@ -1,4 +1,5 @@
 <button class="o-buttons o-buttons--primary">Primary</button>
 <button class="o-buttons o-buttons--primary o-buttons--big">Primary</button>
+<button class="o-buttons o-buttons--primary o-buttons-icon o-buttons-icon--arrow-right o-buttons--big">Primary</button>
```

Comment:
This is a personal thing, but the arrow-right icon feels awkward in a button,  don't we usually put a + or a check mark on it?

Reply:
Think the + was an old "add to myft" button which is no longer going to be made using o-buttons. We seen to use a right arrow to indicate an onward journey in places so I'll stick with this demo https://github.com/Financial-Times/n-myft-ui/blob/fd20a62132b68b4a8dae881d58ffaa3e11d8c5eb/components/onboarding-cta/main.scss#L1